src/live-connect.js

```diff
@@ -88,25 +92,30 @@ export function LiveConnect (liveConnectConfig) {
   }
 
   try {
-    const reducer = (accumulator, func) => accumulator.combineWith(func(accumulator.data))
+    const storageHandler = StorageHandler(configuration.storageStrategy, externalStorageHandler)
```

Comment:
we pass both `configuration` and `externalStorageHandler` in the constructor. Should we wrap the `storage.js` into an object and pass it instead of the `configuration.storageStrategy` if the `configuration.storageStrategy==null`?

Reply:
I don't get it... Storage.js is just our implementation  of the StorageHandler, and already fits the signature, so IMHO there's no need to wrap anything there.